source/agora/common/EnrollmentManager.d

```diff
@@ -443,4 +515,12 @@ unittest
 
     // test for getEnrollment with removed enrollment
     assert(!man.getEnrollment(utxo_hash2, stored_enroll));
+
+    // test for enrollment block height update
+    assert(!man.getEnrolledHeight(utxo_hash));
+    assert(man.updateEnrolledHeight(utxo_hash, 1));
+    assert(man.getEnrolledHeight(utxo_hash));
```

Comment:
Check the actual value

Reply:
applied the suggestion.

Reply:
Did you push ?

Reply:
Yes, Why not display `Outdated`

Reply:
I did it.

Reply:
What Mathias meant is to check the return value. so `assert(man.getEnrolledHeight(utxo_hash) == 8);`, or whichever the right height is supposed to be.

Reply:
I overlooked the obvious. 
I Modified it.